### ISSUE TITLE
From BYOL/SimSiam to DINO with ResNet Backbone

### ISSUE THREAD

**Issue by tshrjn** [lightly-ai/lightly#391]
Are there any plans to implement DINO with at least the ResNet backbone?

**Comment by IgorSusmelj** [comment]
Hi @tshrjn, that's a great idea. Not planned yet but if we have time we can look into it in June. We're currently doing a refactoring of the models. After that, we can look into DINO.
If you are interested in starting with the implementation yourself and contributing to lightly we would be super happy to help you start with the implementation earlier :)

As a first step, you could add more information about DINO and how we should implement it to this issue. That would help already a lot. Maybe you already have an idea?

**Comment by philippmwirth** [comment]
#502 will add the required multi-crop collate functions and with #494 we will have a more flexible implementation of the momentum updates. Since the loss function is simply the cross-entropy between the "feature" vectors, we should be able to easily implement an example for DINO soon 👍

**Comment by philippmwirth** [comment]
This has been closed by #699 and #704.
